### PR TITLE
[#139696025] Complete rendering related photos on PhotoDetails page.

### PR DESCRIPTION
**Pull request by clearlyTHUYDOAN**
**Major**
- New state: tag. Holds the first tag that comes back from getPhotoDetails().
- getPhotoDetails() action uses thunk to call getRelatedPhotos with the tag state (const tag = getState().photos.tag;) after the initial response.
> getRelatedPhotos is called with the first tag that comes back from the response.
- RelatedPhotoGallery is passed state.photos.relatedList (array of photos that comes back from getRelatedPhotos()) and maps it to render Images > Image in a grid below the comments. 

**Minor**
- Changed all function prop types to .isRequired.

### CODE REVIEW

**Comment by clearlyTHUYDOAN** [comment]
I guess because I continued working on my branch after it was merged, it's showing all my previous commits that were dealt with. Should I just remove this pull request, make a new branch, and then push that? Or is that just the normal thing for those to show. Let me know. @melaniebrgr @merlinex

**Comment by clearlyTHUYDOAN** [comment]
Continuing with that thought, it's not that many things but I can make a new branch and push if you think that's cleaner @merlinex . If we keep this, I think the same circleci tests are failing that we looked at @melaniebrgr.

**Comment by melaniebrgr** [comment]
Yeah, these failing tests were fixed and merged into @merlinex's PR, so they _should_ be fixed for this one if you rebase.

**Comment by melaniebrgr** [comment]
Remember, you can check to make sure your tests pass before pushing by running `npm run test` in terminal.

File: src/actions/index.js [hunk 1]
```diff
@@ -127,16 +127,21 @@ export function getCurrentLocation() {
 }
 
 export function getPhotoDetails(params) {
-  return dispatch => {
+  return (dispatch, getState) => {
```
Comment (melaniebrgr) [inline src/actions/index.js:130]: getState is passed as an argument by redux-thunk? Useful!

File: src/components/Image/Image.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import { Link } from 'react-router';
 
 import './Image.css';
 
+import './Image.css';
```
Comment (melaniebrgr) [inline src/components/Image/Image.js]: Imported twice?

**Comment by melaniebrgr** [comment]
I think I introduced some problems while rebasing this branch. 82deb78 makes fixes to get relatedPhotoGallery working again.

**Review by melaniebrgr — APPROVED** [review]

**Pull request merged**